src/casp/services/db/models/users.py

```diff
@@ -12,7 +12,7 @@ class User(db.Base):
     cells_processed = sa.Column(sa.Integer, default=0, nullable=False)
     is_admin = sa.Column(sa.Boolean(), default=True, nullable=False)
 
-    __tablename__ = "user"
+    __tablename__ = "users_user"
```

Comment:
Sorry, if you're renaming, should this just be `users`?

Reply:
Let me give you some context and motivation behind it. The naming convention I've adopted was inspired by the database table naming policy from the Django framework. The format I use is 'db_model_module_ormModelName'. This approach allows us to easily structure tables by names and application modules. Our current application module is 'users' (where all user-related models go), and the model name is 'User'; thus, the table name is 'users_user'. I thought it would be useful to incorporate table prefixes. This way, when you add your new model to auth-related stuff (e.g., Token or ServiceAccount), the table name appears as 'users_token' or 'users_serviceaccount', respectively. Also it helps to organize tables when listing them from data grips (screenshot attached). Please let me know what you think about it.
<img width="600" alt="image" src="https://github.com/cellarium-ai/cellarium-cloud/assets/26435815/9e8e7904-5932-4d24-b79e-49dfef6d008b">

Reply:
ah ok.  I got you.  I think that's fine.  And it's definitely better than `user` which was causing syntax errors in SQL if left unquoted